docs/advtool-installation.html

```diff
@@ -191,11 +192,10 @@ <h4 class="ibm-h4">Taking advantage of leading edge tools and libraries</h4>
     <li>The gpg public key <code>gpg-pubkey-6976a827-5164221b</code> is provided in the University of Campinas (Unicamp) repositories. This public key is used to verify the authenticity of both the Advance Toolchain packages and the repository contents. Download the gpg public key for your Linux distribution and import it using the following commands:
       <pre>    wget ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/redhat/RHEL7/gpg-pubkey-6976a827-5164221b
     rpm --import gpg-pubkey-6976a827-5164221b</pre>
-      <p>Notes:
+      <p>Notes:</p>
         <ul>
           <li>If you are installing on Red Hat 6, you must include two gpg public keys: <code>gpg-pubkey-6976a827-5164221b</code> and <code>gpg-pubkey-3052930d-5175955a</code>.</li>
         </ul>
-      </p>
```

Comment:
@er-1 Are you sure this change will generate the exact same output?
I remember writing HTML like this one and having issues to guarantee the `<li>` entity would be properly indented, e.g. I expected to see the text "If you are installing..." to have more indentation than "Notes:".

Reply:
@tuliom I checked: [before](https://i.ibb.co/YXBggsx/before.png) [after](https://i.ibb.co/pJMZq2h/after.png)
Same output.

Reply:
Thanks for clarifying this!